`scalingo notifiers`: crashes if no-event notifier exists

It is possible for "no-event" notifiers to exist on applications.

With Scalingo CLI 1.24.2, when displaying the list of notifiers for an application that has such a notifier, the CLI crashes.

## How to reproduce

1. Create a no-event notifier on the current app:
```
$ scalingo notifiers-add --name test --platform email
+-----------------+-----------------------------------------+
| ID              | …                                       |
| Type            | email                                   |
| Name            | test                                    |
| Enabled         | true                                    |
| Send all events | false                                   |
| Emails          | []                                      |
| User_ids        | []                                      |
| Selected events |                                         |
+-----------------+-----------------------------------------+
-----> Notifier have been created.
```
(btw there's a typo here, it should read `Notifier has been created`).

2. Try to list the notifiers of the app with `scalingo notifiers`:
```
$ scalingo notifiers
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/Scalingo/cli/notifiers.eventTypesToString({0x1400058d000, 0x25, 0x2a}, {0x103706588, 0x0, 0x0})
	/home/runner/work/cli/cli/notifiers/list.go:62 +0x180
github.com/Scalingo/cli/notifiers.List({0x16d2337e4, 0x12})
	/home/runner/work/cli/cli/notifiers/list.go:36 +0x31c
github.com/Scalingo/cli/cmd.glob..func133(0x14000160c60)
	/home/runner/work/cli/cli/cmd/notifiers.go:27 +0x4c
github.com/Scalingo/cli/cmd.(*AppCommands).addCommand.func1(0x14000160c60)
	/home/runner/work/cli/cli/cmd/commands.go:67 +0x2ec
github.com/urfave/cli.HandleAction({0x1031ff7c0, 0x140002262d0}, 0x14000160c60)
	/home/runner/work/cli/cli/vendor/github.com/urfave/cli/app.go:526 +0x60
github.com/urfave/cli.Command.Run({{0x1030decaa, 0x9}, {0x0, 0x0}, {0x0, 0x0, 0x0}, {0x1030e663f, 0x13}, {0x0, ...}, ...}, ...)
	/home/runner/work/cli/cli/vendor/github.com/urfave/cli/command.go:173 +0x610
github.com/urfave/cli.(*App).Run(0x140001d2700, {0x14000132040, 0x4, 0x4})
	/home/runner/work/cli/cli/vendor/github.com/urfave/cli/app.go:277 +0x60c
main.main()
	/home/runner/work/cli/cli/scalingo/main.go:158 +0x648
```

Expected behavior: show the list of notifiers.

--- a/notifiers/create.go
+++ b/notifiers/create.go
@@ -69,7 +69,7 @@ func Provision(ctx context.Context, app, platformName string, params ProvisionPa
 	displayDetails(notifier, eventTypes)
 
 	io.Info()
-	io.Status("Notifier have been created.")
+	io.Status("Notifier has been created.")
 	return nil
 }
 

--- a/notifiers/list.go
+++ b/notifiers/list.go
@@ -47,10 +47,11 @@ func List(ctx context.Context, app string) error {
 	return nil
 }
 
-func eventTypesToString(eventTypes []scalingo.EventType, ids []string) (res string) {
+func eventTypesToString(eventTypes []scalingo.EventType, ids []string) string {
+	res := ""
 	if len(ids) == 0 {
 		res = ""
-		return
+		return res
 	}
 	switch len(eventTypes) {
 	case 0:
@@ -70,5 +71,5 @@ func eventTypesToString(eventTypes []scalingo.EventType, ids []string) (res stri
 			}
 		}
 	}
-	return
+	return res
 }

--- a/notifiers/list.go
+++ b/notifiers/list.go
@@ -48,6 +48,10 @@ func List(ctx context.Context, app string) error {
 }
 
 func eventTypesToString(eventTypes []scalingo.EventType, ids []string) (res string) {
+	if len(ids) == 0 {
+		res = ""
+		return
+	}
 	switch len(eventTypes) {
 	case 0:
 		res = ""

